packages/manager/apps/web/client/app/domain/webhosting/order/steps/hosting/domain-webhosting-order-steps-hosting.html

```diff
@@ -0,0 +1,101 @@
+                            </dd>
+                        </div>
+                        <div
+                            data-ng-if=":: offer.sharedDatabases.length > 0"
+                            data-ng-repeat="shared in ::offer.sharedDatabases track by $index">
+                            <dt
+                                data-ng-if="$first"
+                                data-translate="domain_webhosting_order_hosting_step_option_shared_database"></dt>
+                            <dd>
+                                <span data-translate="{{:: 'domain_webhosting_order_hosting_step_option_shared_database_' + shared.type }}"></span>
+                                <span data-ng-bind="::shared.available + ' x ' + shared.value"></span>
+                                <span data-translate="{{:: 'domain_webhosting_order_hosting_step_option_size_unit_' + shared.unit | translate }}"></span>
+                            </dd>
+                        </div>
+                        <dt data-translate="domain_webhosting_order_hosting_step_option_private_database"></dt>
+                        <dd>
+                            <span
+                                data-ng-if=":: offer.privateDatabases.length === 0"
+                                data-translate="domain_webhosting_order_hosting_step_option_private_database_empty"></span>
+                            <span
+                                data-ng-repeat="private in ::offer.privateDatabases track by $index"
+                                data-ng-if=":: offer.privateDatabases.length > 0">
+                                <span
+                                    data-ng-if="$first"
+                                    data-translate="domain_webhosting_order_hosting_step_option_private_database_existing"></span>
+                                <span data-ng-bind=":: private.quota.value"></span>
+                                <span data-translate="{{:: 'domain_webhosting_order_hosting_step_option_size_unit_' + private.quota.unit}}"></span>
+                            </span>
+                        </dd>
+                    </dl>
+                    <a
+                        class="oui-link_icon ml-4"
+                        target="_blank"
+                        rel="noopener"
+                        href="{{:: offer.guideUrl }}">
+                        <span data-translate="domain_webhosting_order_hosting_step_option_find_out_more"></span>
+                        <span
+                            class="oui-icon oui-icon-external_link"
+                            aria-hidden="true"></span>
+                        <span
+                            class="sr-only"
+                            data-translate="domain_webhosting_order_hosting_step_option_open_new_tab"></span>
+                    </a>
+                </oui-select-picker-section>
+                <oui-select-picker-footer>
+                    <order-catalog-price
```

Comment:
The component name is a bit to generic in my opinion.

What's do you think about specifying at least the universe?
Because actually this component lives only in the Web Control Panel.

Component location: `packages/manager/apps/web/client/app/components/order-catalog-price/index.js`.

Reply:
Hmm in fact, this component could be exported in a upper scope than just web.
As we race against the time, for the moment, this component is located into web/components folder.

But in long term, it should be available for all of the manager scope. So to simplify, this component is named `order-xxx` (like its siblings components).

And now you will say, "but instead we could prefix `manager-order-xxx` isn't it ? :p

Reply:
@JeremyDec prefix seems OK in that case.

Maybe you can consider using `<ovh-manager-order-...>`?

Is that make sense for you?

Reply:
Yes I agree :) @AxelPeter => I'll make the change on feat/miaw so you will be able to get it